src/XMakeBuildEngine/Construction/ProjectRootElement.cs

```diff
@@ -655,6 +650,28 @@ public string InitialTargets
         }
 
         /// <summary>
+        /// Gets or sets a semicolon delimited list of software development kits (SDK) that the project uses.
+        /// If  a value is specified, an Sdk.props is simplicity imported at the top of the project and an
+        /// Sdk.targets is simplicity imported at the bottom from the specified SDK.
+        /// If the value is null or empty, removes the attribute.
+        /// </summary>
+        public string Sdk
+        {
+            [DebuggerStepThrough]
+            get
+            {
+                return ProjectXmlUtilities.GetAttributeValue(XmlElement, XMakeAttributes.sdk);
```

Comment:
As things get more involved with the SDKs, it might be worth creating a new SDKProjectElement which is represented as an attribute (reusing the way metadata as attributes represent themselves). This type would be a valid child for ProjectRootElements and ProjectImportElements. Then, the new type could gain responsibilities such as parsing the semicolon delimited string into a {name, version} object, doing data validation, etc